stable/awsebscsiprovisioner/values.yaml

```diff
@@ -64,7 +64,7 @@ storageclass:
   isDefault: false
   reclaimPolicy: Delete
   volumeBindingMode: WaitForFirstConsumer
-  type: gp2
+  type: io1
```

Comment:
Could we put a comment here with our reasoning for using io1? Might just be a nice breadcrumb for future code readers to better understand our configuration.